doc/architecture/squid.md

```diff
@@ -700,6 +872,25 @@ Example
 ocean.agreements.conditions.grantAccess(agreementId, assetId, grantee)
 ```
 
+#### grantServiceExecution
```

Comment:
Since this has the same params and return types as `grantAccess` maybe it is not necessary to have this separate api.